README.md

```diff
@@ -20,23 +21,21 @@ Instructions for other flavors and other OSs will be provided in the future.
 4. run `sudo apt-get install zlib1g-dev`
 5. run `sudo apt-get install libbz2-dev`
 6. run `sudo apt-get install doxygen`
+7. run `sudo apt-get install stormlib`
```

Comment:
stormlib appears to be a third-party package. Please add a note along these lines: If the stormlib package cannot be found got to [this](http://xpam.pl/blog/?p=117) website and follow the instructions for deb based distros.